src/teaching-element/Assessment/NumericalResponse.vue

```diff
@@ -50,8 +50,10 @@ export default {
       });
     },
     update() {
-      const userAnswer = map(this.items, it => toNumber(it.answer));
-      const isValid = every(userAnswer, it => it && !isNaN(it));
+      const userAnswer = map(this.items, ({ answer }) => {
+        return answer === '' ? NaN : toNumber(answer);
+      });
+      const isValid = every(userAnswer, it => !isNaN(it));
```

Comment:
```js
// in top level scope:
const toNumber = str => str.trim() ? 0 : Number(str);

// later on inside component...
update() {
  const userAnswer = this.items.map(({ answer }) => toNumber(answer));
  const isValid = userAnswer.items(it => Number.isNaN(it));
  this.$emit('validateAnswer', { isValid });
  this.$emit('update', { userAnswer });
}
```

1. You know `items` is an array so you could use native `map`
2. `userAnswer` is definitely an array so you could use native `every`
3. `_.toNumber` works exactly the same as `Number()` constructor
4. there is native method for checking `NaN`s: `Number.isNaN`

Result: bundle is lighter due to less lodash utilities being brought in 😉